tests/playwright/src/model/pages/agent-workspace-terminal-page.ts

```
@@ -0,0 +1,82 @@
+        message: `Typed text "${prompt}" did not appear in the terminal`,
+      })
+      .not.toBe(contentBefore);
+
+    await this.page.keyboard.press('Enter');
+
+    if (expectedResponse) {
+      const contentAfterSubmit = await this.getTerminalText();
+      await expect
+        .poll(() => this.getTerminalText(), {
+          timeout,
+          message: `Terminal content did not change after pressing Enter`,
```

Biome (2.4.15)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)